applications/serial_lte_modem/src/slm_at_udp_proxy.c

```diff
@@ -481,7 +529,7 @@ int handle_at_udp_server(enum at_cmd_type cmd_type)
 }
 
 /**@brief handle AT#XUDPCLI commands
- *  AT#XUDPCLI=<op>[,<url>,<port>[,<sec_tag>]
+ *  AT#XUDPCLI=<op>[,<url>,<port>[,<sec_tag>,<connection_id>]
```

Comment:
```suggestion
 *  AT#XUDPCLI=<op>[,<url>,<port>[,<sec_tag>[,<connection_id>]
```

Connection ID parameter seem optional.

Reply:
You're right, it's optional.